### DISCUSSION TITLE
docs/ninjutsu/dark-ninja/zip-bomb

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#29] [General]
# docs/ninjutsu/dark-ninja/zip-bomb

该摸鱼忍术只是为了图一乐，切勿在工作设备中操作!

https://996.ninja/docs/ninjutsu/dark-ninja/zip-bomb

**Comment by pointm** [comment]
如果WINDOWS无法运行这个赏心悦目的FORK炸弹
```
:(){ :|:& };:
```
可以安装完GIT之后，右键桌面选择`Open Git Bash Here`，粘贴到窗口里面后回车，让Git在后台运行即可，效果相同[doge]。解法就是重启即可。

**Reply by MingGH** [reply]
鹅国jio本有两把刷子！

**Reply by pointm** [reply]
> 鹅国jio本有两把刷子！

身为JIO本，那当然得深谙摸鱼忍术才行
